VideoWeb/VideoWeb/ClientApp/src/app/services/logging/loggers/app-insights-logger.service.ts

```diff
@@ -124,7 +124,7 @@ export class AppInsightsLoggerService implements LogAdapter {
             .pipe(filter(Boolean))
             .subscribe(() => {
                 this.profileService.getUserProfile().then(profile => {
-                    this.isVHO = profile.role === Role.VideoHearingsOfficer;
+                    this.isVHO = profile.roles?.includes(Role.VideoHearingsOfficer);
```

Comment:
roles shouldn't be null. if it is maybe confirm it's always instantiated as an empty array in the ctor in the Contract DTO so it doesn't serialise into a null